src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1589,6 +1597,33 @@ public final void testDeletePatronAccountByIdItemByItemIdHoldByHoldIdWithErrors(
     logger.info("Test done");
   }
 
+  @Test
+  public final void allowedServicePointsShouldSucceed() {
+    logger.info("Testing allowed service points");
+
+    var allowedSpResponse = given()
+      .header(tenantHeader)
+      .header(urlHeader)
+      .header(contentTypeHeader)
+      .pathParam("accountId", goodUserId)
+      .pathParam("instanceId", goodInstanceId)
+      .log().all()
+      .when()
+      .contentType(ContentType.JSON)
+      .get(accountPath + instancePath + allowedServicePointsPath)
+      .then()
+      .log().all()
+      .and().assertThat().contentType(ContentType.JSON)
+      .and().assertThat().statusCode(200)
+      .extract()
+      .asString();
+
+    final JsonObject expectedJson = new JsonObject(readMockFile(mockDataFolder +
+      "/allowed_sp_mod_patron_expected_response.json"));
+    verifyAllowedServicePoints(expectedJson, new JsonObject(allowedSpResponse));
+    logger.info("Test done");
```

Comment:
do we really need this logging message?

Reply:
these logs are in all of the other tests..